SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Thunderbird for Android Security
+
+## Security Audit
+
+The code in this repository was undergoing an extensive security audit in collaboration with the Open Source Technology
```

Comment:
Should this say "underwent an extensive security audit"?

Reply:
Good catch, fixed